app/lib/theme/getFilteredSchema.ts

```diff
@@ -0,0 +1,19 @@
+import type { JSONSchema7 } from "json-schema";
+
+// Filter out fields from the formSchema that have not changed from the previous revision so the summary ignores these fields
+// This is mainly used when the form has multiple fields within it and we want to check each field data with the previous revision
+export const getFilteredSchema = (formSchema: JSONSchema7, formData) => {
```

Comment:
Can this replace functionality in any of the other summary components?

Reply:
`ProjectFormSummary` is using similar functionality (lines 84 to 103).